.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,51 @@
+name: Acceptance tests
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  acceptance:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        setfile:
+          - centos7-64{hostname=centos7-64.example.com}
+          - centos8-64{hostname=centos8-64.example.com}
+        puppet:
+          - "6"
+          - "5"
+        pulpcore:
+          - '3.4'
+          - '3.6'
+          - '3.7'
```

Comment:
This implies that the development branch is attempting to maintain support for 3 versions of Pulp at a time. Is that are true desire? That seems more than standard.

Reply:
It is what we now state in the README. I'm not sure where we should land, but I think this is a pattern I want to apply in more places, such as puppet-foreman.

Reply:
The thing I want to make sure we think about is how much choosing to support some amount of previous versions could pigeon hole us into additional work down the road trying to keep the main branch compatible across versions (vs. utilizing module releases and stable branches to achieve that).